File: idling-registry-okhttp/app/src/androidTest/java/com/sqisland/espresso/idling_registry/okhttp/MainActivityTest.java
package com.sqisland.espresso.idling_registry.okhttp;

import android.support.test.rule.ActivityTestRule;

import org.junit.Rule;
import org.junit.Test;

import static android.support.test.espresso.Espresso.onView;
import static android.support.test.espresso.assertion.ViewAssertions.matches;
import static android.support.test.espresso.matcher.ViewMatchers.withId;
import static android.support.test.espresso.matcher.ViewMatchers.withText;

public class MainActivityTest {
  @Rule
  public ActivityTestRule<MainActivity> activityRule = new ActivityTestRule<>(MainActivity.class);

  @Test
  public void name() {
    onView(withId(R.id.name))
        .check(matches(withText("Chiu-Ki Chan")));
  }
}

File: idling-registry-okhttp/app/src/androidTest/java/com/sqisland/espresso/idling_registry/okhttp/MainActivityTest.kt
package com.sqisland.espresso.idling_registry.okhttp

import android.support.test.espresso.Espresso.onView
import android.support.test.espresso.assertion.ViewAssertions.matches
import android.support.test.espresso.matcher.ViewMatchers.withId
import android.support.test.espresso.matcher.ViewMatchers.withText
import android.support.test.rule.ActivityTestRule
import org.junit.Rule
import org.junit.Test

class MainActivityTest {
  @get:Rule
  val activityRule = ActivityTestRule(MainActivity::class.java)

  @Test
  fun name() {
    onView(withId(R.id.name))
      .check(matches(withText("Chiu-Ki Chan")))
  }
}